examples/plot_2D_simulation_example.py

```diff
@@ -295,11 +297,7 @@ def plot(maps, titles):
 # ensemble of clustered desparsified lasso (EnCluDL)
 list_ward, list_beta_hat, list_theta_hat, list_omega_diag = (
     ensemble_clustered_inference(
-        X_init,
-        y,
-        ward,
-        n_clusters,
-        scaler_sampling=StandardScaler(),
+        X_init, y, ward, n_clusters, scaler_sampling=StandardScaler(), seed=73
     )
 )
```

Comment:
I've just checked, and it turns out that the example is as good if you set all seeds to 0...

Reply:
For me, I prefer the example before because desparsified also has some detection of regions and CluDL is much worse than EnCluDL.

Before:
<img width="400" height="600" alt="Figure_1_3" src="https://github.com/user-attachments/assets/201ab3ed-3a18-40d5-be1b-465c62f3a626" />
After:
<img width="400" height="600" alt="Figure_1_4" src="https://github.com/user-attachments/assets/db2c1eea-37b2-44bc-b9dd-4793570072da" />

Reply:
But it is indeed possible that desparsified  is powerless, which is seen in both cases. 
We also see in both cases that EnCluDL is slightly more regular than CluDL, as expected.
IMHO, the difference in the result is not meaningful enough that we should "tune" the seed.